app/helpers/check_functions.py

```diff
@@ -88,3 +89,20 @@ def get_and_check_icon(icon_set, icon_name):
         logger.error("The icon doesn't exist: %s", path)
         abort(404, "Icon not found in icon set")
     return icon
+
+
+def check_if_descripton_file_exists(icon_set):
+    """
+    Checks that the icon set has a corresponding dictionary containing description for all available
+    languages.
+    if not raises a flask error and abort the current request.
+
+    Args:
+        icon_set: (IconSet) the icon set in which belongs the icon we want
+        icon_name: (str) the name of the icon
```

Comment:
Currently only `icon_set` is an argument

Reply:
tbd @pakb @LukasJoss : Do we want to check the existence of keys in the dict for all languages here, or in the front-end? Might make sense to have that check here

Reply:
Probably front-end could handle the case when a certain language is missing in the returned dict, by simply skipping the mouseover text in that case

Reply:
We would probably need to define first which languages are required in the first place, but handling it on the front-end is probably fine